inform7/Internal/Inter/Architecture32Kit/Sections/Input Output.i6t

```diff
@@ -319,6 +322,17 @@ to document all of that.
         glk_put_char_stream(gg_commandstr, 10); ! newline
     }
   .KPContinue;
+
+    uwl = UnicodeWhitespace + WORDSIZE;
```

Comment:
Doesn't this mean it skips 133?

Reply:
yeah, remnant of when the length was the first element.